src/core/gltf2exporter/dracocompressor_p.h

```diff
@@ -0,0 +1,53 @@
+/*
+    dracocompressor_p.h
+
+    This file is part of Kuesa.
+
+    Copyright (C) 2019 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+    Author: Juan Jose Casafranca <juan.casafranca@kdab.com>
+
+    Licensees holding valid proprietary KDAB Kuesa licenses may use this file in
+    accordance with the Kuesa Enterprise License Agreement provided with the Software in the
+    LICENSE.KUESA.ENTERPRISE file.
+
+    Contact info@kdab.com if any conditions of this licensing are not clear to you.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as
+    published by the Free Software Foundation, either version 3 of the
+    License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef DRACOCOMPRESSOR_H
+#define DRACOCOMPRESSOR_H
+
+#include <memory>
+#include <draco/compression/encode.h>
+#include <vector>
+#include <QString>
+
+namespace Qt3DRender {
+class QGeometry;
+}
+namespace Kuesa {
+class GLTF2ExportConfiguration;
+namespace Compression {
+
+struct CompressedMesh {
```

Comment:
{ should be on new line for structs and classes I believe

Reply:
hmm, when I run clang-format on that file (with `clang-format -i dracocompressor_p.h`)  it puts the brace here

Reply:
yes, structs and classes have different rules. structs are treated like enums regarding that point.

Reply:
I added a destructor to that struct (in order to not require a draco header in that file) - should I keep it as a struct or change it to a class ?

Reply:
done

Reply:
I'd say class in such a case. But I don't see the aforementioned destructor anyway.